doc/cs_registers.rst

```diff
@@ -96,6 +100,30 @@ CSR Address: ``0x301``
 On Ibex, ``misa`` is hard-wired, i.e. it will remain unchanged after any write.
 
 
+Machine Interrupt Enable Register (mie)
+---------------------------------------
+
+CSR Address: ``0x304``
+
+Reset Value: ``0x0000_0000``
+
+``mie`` is a WARL register which allows to individually enable/disable local interrupts.
+After reset, all interrupts are disabled.
+
++-------+--------------------------------------------------------------------------------------+
+| Bit#  | Interrupt                                                                            |
++-------+--------------------------------------------------------------------------------------+
+| 30:16 | Machine Fast Interrupt Enables: Set bit x+16 to enable                               |
+|       | fast interrupt ``irq_fast_i[x]``.                                                    |
```

Comment:
The NMI logic needs to be documented here as well.

Reply:
According to the spec, the NMI is not visible in the `mip` CSR. Thus I did not include it here.